HotKey/HotKey_MainFolder/HotKeyItem.cs

```diff
@@ -9,26 +9,84 @@ namespace HotKey_MainFolder
 {
     public class HotKeyItem
     {
+        private KeyboardHook hook;
+        private Dictionary<Tuple<ModKeys, Keys>, Action> modeFormKeybindDictionary;
+        private Action action;
         private string actionName;
         private ModKeys modKeys;
-        private Keys keys;
+        private Keys key;
 
-        public HotKeyItem() { }
-
-        public HotKeyItem(string actionName, ModKeys modKeys, Keys keys)
+        public HotKeyItem(KeyboardHook hook, Dictionary<Tuple<ModKeys, Keys>, Action> modeFormKeybindDictionary, Action action, string actionName, ModKeys modKeys, Keys key)
```

Comment:
Just my own personal preference is when something gets this long is to move it down to the next line.